### PR TITLE
Notifications: Custom ReaderPostDetails Title

### LINKED ISSUE #3373: A post detail screen accessed from the notifications feature shouldn't be titled "Reader"
It might be a bit confusing to be on the notifications tab and see a screen titled "Reader" while the post content is loading. 

Maybe we can just say "Post" or even leave the title blank while loading? 

cc @jleandroperez

### PR DESCRIPTION
**Pull request by jleandroperez**
Closes #3373

/cc @aerych (thank you sir!)

### CODE REVIEW

**Comment by aerych** [comment]
Patch looks good, but I'm wondering when we'd want to use "reader" as a placeholder and not "post". For any detail opened from the reader, the post is already loaded and the title is immediately available so a title placeholder won't be used.  Does it make sense to just make the placeholder "Post" and not worry about setting the title from the opening VC?

**Comment by jleandroperez** [comment]
@aerych you're right, as usual!. I didn't know we actually wanted to update the default Title (sorry about that!).

PR simplified and ready for a quick review.

Thanks!

File: WordPress/Classes/ViewRelated/Reader/ReaderPostDetailViewController.m [hunk 1]
```diff
@@ -294,7 +294,8 @@ - (void)dismissPopover
 
 - (void)refresh
 {
-    self.title = self.post.postTitle ?: NSLocalizedString(@"Reader", @"Placeholder title for ReaderPostDetails.");
+    // Allow the Title Placeholder to be set externally. If empty, fall back to "Reader"
```
Comment (aerych) [inline WordPress/Classes/ViewRelated/Reader/ReaderPostDetailViewController.m]: > If empty, fall back to "Reader"

Post?

**Comment by aerych** [comment]
Shiney!   :shipit:

**Pull request merged**